tests/test_integration.py

```diff
@@ -455,6 +457,36 @@ def test_info(self):
         self.vm.shutdown()
         host_info(VM_HOSTNAME)
 
+    @patch('igvm.vm.VM.vm_performance_value', return_value=5.0)
+    def test_igvm_migration_log(self, mock_vm_performance_value):
+        self.vm.hypervisor.dataset_obj['igvm_migration_log'].clear()
+        self.vm.hypervisor.dataset_obj.commit()
+        self.assertFalse(
+            bool(self.vm.hypervisor.dataset_obj['igvm_migration_log'])
+        )
+
+        cpu_usage_vm = self.vm.hypervisor.hv_predict_vm_cpu_util(self.vm)
+        timestamp = int(time())
+        self.vm.hypervisor.hv_add_migration_log(self.vm, '-')
```

Comment:
unfortunately this does not make any sense now. we want to test that `vm_migrate()` add this entry. this test will always pass now. don't write the migration log in the test. do the migration and then do the `assertEqual`.
same applies for the other `hv_add_migration_log()` below.
save `self.vm.hypervisor` into some `prev_hv` variable because the `vm_migrate()` call will overwrite it. then you can do your asserts on `prev_hv` and `self.vm.hypervisor`

Reply:
oh and to circumvent the problem with the timestamp you can mock the current time as well

Reply:
I reworked it again.
For whatever reason the VM object isn't reinitialized after the migration, so I had to workaround it a bit by getting a fresh object with `_get_vm(VM_HOSTNAME)`

Anyway, only the migration is now manipulating the `igvm_migration_log` and the asserts are working as expected.